tools/docs/manager.ts

```diff
@@ -73,6 +160,24 @@ sidebar_label: ${displayName}
     }
     md += managerReadmeContent + '\n\n';
 
+    const [bugList, bugsLen] = stringifyIssues(managerIssuesMap[manager]?.bugs);
+    if (bugList) {
+      md += '<!-- prettier-ignore -->\n';
+      md += `??? note "Click me to see the list of ${bugsLen} open bug reports"\n`;
```

Comment:
nit: I don't think the number of bugs adds much. Feel free to ignore if you disagree

Reply:
I'm not sure about the "Click me" bit too. @HonkingGoose I think this came from you, first for logs in our bug reporting. Do you think in this case we also need to explain to people "click me"?

Reply:
An in between solution could be as suggested [here](https://github.com/renovatebot/renovate/issues/15546#issuecomment-1140790514) by @betterPeleg. 
We can show first few issues and collapse the rest.

Reply:
> nit: I don't think the number of bugs adds much. Feel free to ignore if you disagree

The idea behind having the numbers was to "show" to our readers that the collapsible section has items in it. Readers will then know that they can expand the collapsed section to see the full list.

> I'm not sure about the "Click me" bit too. @HonkingGoose I think this came from you, first for logs in our bug reporting. Do you think in this case we also need to explain to people "click me"?

We can probably drop the "Click me" text, and rely on the visual style and title of the admonition to show that the section can be expanded. [^mat-docs]

> An in between solution could be as suggested https://github.com/renovatebot/renovate/issues/15546#issuecomment-1140790514 by @betterPeleg.
> We can show first few issues and collapse the rest.

So we would show the most recent issues first, and then let the readers check out the older issues by expanding the collapsed admonition. I don't know if this will look nice or not. It might make the sections look odd visually. The admonitions have a lot of visual weight compared to the boring lists. We can always try it out first, and later adjust.

[^mat-docs]: [Material for MkDocs documentation, Collapsible blocks](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks)

Reply:
> The idea behind having the numbers was to "show" to our readers that the collapsible section has items in it. Readers will then know that they can expand the collapsed section to see the full list.

Good point. I think my main concern, as a user, would be to see a large number and think that manager is not well maintained. In reality, the opposite may be true. It's a well used, and well maintained manager, that is exploring lots of corner cases!

Happy to keep the numbers though, as a visual indicator that the item is expandable.

Reply:
Resolving this thread as it's no longer relevant.